presentations/2024-10-10_what-is-ai-chris/index.qmd

```diff
@@ -0,0 +1,162 @@
+
+## The "AI effect"
+
+> "The AI effect" refers to a phenomenon where either the definition of AI or the concept of intelligence is adjusted to exclude capabilities that AI systems have mastered. This often manifests as tasks that AI can now perform successfully no longer being considered part of AI, or as the notion of intelligence itself being redefined to exclude AI achievements ([wikipedia](https://en.wikipedia.org/wiki/AI_effect))
+
+## The coffee test
+
+"A machine is required to enter an average American home and figure out how to make coffee: find the coffee machine, find the coffee, add water, find a mug, and brew the coffee by pushing the proper buttons." ([source](https://en.wikipedia.org/wiki/Artificial_general_intelligence#Characteristics))
+
+## A narrower definition of "AI" for us to use
+
+* Searching the possibility space of the game of chess is not AI- because it doesn't scale
+* A robot coming to your house and making a cup of coffee **is** AI but it's way more sophisticated than we need
+* Useful AI systems today are basically something inbetween
+
+## Machine learning
+
+![](ml.png)
+
+## Right, so, really, what is AI?
+
+* Classification is often called "AI"
+* Various types of classification problems:
+  * Given a set of observations will this patient require hospitalisation?
+  * Given a piece of patient feedback which of these 10 categories is it about?
+  * Given an X ray is it probable that this patient has cancer?
+
+## No, really, tell me, what is AI?
+
+* There are two elements of a classification problem that can be "difficult"
+  * Input
+  * Computation
+* I'd really like to avoid calling anything AI if one or both of those things is not complex
+
+## Risk score
+
+* Sometimes you see any predictive algorithm as "AI"
+
+```{r, echo = FALSE, eval = TRUE}
+
+fit = glm(vs ~ hp, data=mtcars, family=binomial)
+newdat <- data.frame(hp=seq(min(mtcars$hp), max(mtcars$hp),len=100))
+newdat$vs = predict(fit, newdata=newdat, type="response")
+plot(vs~hp, data=mtcars, col="red4", ylab = "Outcome", xlab = "Score")
+lines(vs ~ hp, newdat, col="green4", lwd=2)
+
+```
+
+## Deep learning
+
+* Deep learning is literally artificial intelligence
+
+![](dl.png)
+
+([source](https://blog.bismart.com/en/difference-between-machine-learning-deep-learning))
+
+## Deep learning predictions
+
+* Deep learning models meet a reasonable standard for "AI"
+* They model complex non linear relationships without any prior knowledge, rules, or models (a little bit like a human baby, in fact)
+* Deep learning models for prediction can absorb large numbers of variables and produce predictions based on highly complex relationships within the data, merely by being trained appropriately
+
+## Classifying patient experience feedback
+
+* Complex input
+* For comment theme- non complex computation (logistic regression)
```

Comment:
Does it matter that we didn't actually use a logistic regression model in pxtextmining for theme?

Reply:
I don't think it does but that's a good point- logistic regression worked, that's enough (I think SVMs aren't "intelligent" either but it's not such a clean argument)